aim/cli/runs/utils.py

```diff
@@ -26,3 +30,43 @@ def match_runs(repo_path: str, hashes: List[str]) -> List[str]:
             matched_hashes.add(run_hash)
 
     return list(matched_hashes)
+
+
+
+def make_zip_archive(repo_path: str) -> io.BytesIO:
+    chunks_dir = os.path.join(repo_path, '.aim', 'meta', 'chunks')
```

Comment:
we should create an archive for the entire repo (`.aim`).
runs backup can be added later.